packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/Camera2CameraInfoHostApiImpl.java

```diff
@@ -115,6 +125,19 @@ public Long getSensorOrientation(@NonNull Long identifier) {
     return proxy.getSensorOrientation(getCamera2CameraInfoInstance(identifier));
   }
 
+  @NonNull
+  @Override
+  public List<Long> getAvailableVideoStabilizationModes(@NonNull Long identifier) {
+
+    int[] lst = proxy.getAvailableVideoStabilizationModes(getCamera2CameraInfoInstance(identifier));
```

Comment:
Nit: Can this have a more descriptive name like `availableModes`?

Reply:
Sorry for the confusion--I was referring to `lst`.

Reply:
Of course! It was staring at me and I didn't see it. :-) Changed.